dependency-injection/2.0/TCK-summary.html

```diff
@@ -1,57 +1,58 @@
-[INFO] Configured Artifact: jakarta.inject:jakarta.inject-api:2.0.0-RC4:jar
-[INFO] Configured Artifact: jakarta.inject:jakarta.inject-tck:2.0.0-RC4:jar
-[INFO] jakarta.inject-api-2.0.0-RC4.jar already unpacked.
-[INFO] Unpacking /Users/starksm/.m2/repository/jakarta/inject/jakarta.inject-tck/2.0.0-RC4/jakarta.inject-tck-2.0.0-RC4.jar to /Users/starksm/Dev/JBoss/Jakarta/injection-tck/example/target/test-classes with includes "" and excludes ""
-[INFO] 
+[INFO] Configured Artifact: jakarta.inject:jakarta.inject-api:2.0.0:jar
+[INFO] Configured Artifact: jakarta.inject:jakarta.inject-tck:2.0.0:jar
+[INFO] Unpacking /Users/starksm/.m2/repository/jakarta/inject/jakarta.inject-api/2.0.0/jakarta.inject-api-2.0.0.jar to /Users/starksm/Dev/JBoss/Jakarta/injection-tck/example/target/test-classes with includes "" and excludes ""
+[INFO] jakarta.inject-tck-2.0.0.jar already unpacked.
+[INFO]
 [INFO] --- maven-surefire-plugin:3.0.0-M3:test (default-test) @ jakarta.inject-tck-example ---
-[INFO] 
+[INFO]
 [INFO] -------------------------------------------------------
 [INFO]  T E S T S
 [INFO] -------------------------------------------------------
 [INFO] Running SampleBootstrapTCK
-May 18, 2020 12:40:06 PM org.jboss.weld.bootstrap.WeldStartup <clinit>
+Jul 06, 2020 10:47:23 PM org.jboss.weld.bootstrap.WeldStartup &lt;clinit&gt;
 INFO: WELD-000900: 4.0.0 (Alpha2)
-May 18, 2020 12:40:07 PM org.jboss.weld.bootstrap.WeldStartup startContainer
+Jul 06, 2020 10:47:23 PM org.jboss.weld.bootstrap.WeldStartup startContainer
 INFO: WELD-000101: Transactional services not available. Injection of @Inject UserTransaction not available. Transactional observers will be invoked synchronously.
-May 18, 2020 12:40:07 PM org.jboss.weld.environment.se.WeldContainer fireContainerInitializedEvent
-INFO: WELD-ENV-002003: Weld SE container ba89ce3d-586d-4164-be42-490a50ca9bbf initialized
-[INFO] Tests run: 50, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.484 s - in SampleBootstrapTCK
-Weld SE container ba89ce3d-586d-4164-be42-490a50ca9bbf shut down by shutdown hook
-[INFO] 
+Jul 06, 2020 10:47:23 PM org.jboss.weld.environment.se.WeldContainer fireContainerInitializedEvent
+INFO: WELD-ENV-002003: Weld SE container 8f6770d3-d00a-444d-97ec-b1b7d090512a initialized
+[INFO] Tests run: 50, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.553 s - in SampleBootstrapTCK
+Weld SE container 8f6770d3-d00a-444d-97ec-b1b7d090512a shut down by shutdown hook
+[INFO]
 [INFO] Results:
-[INFO] 
+[INFO]
 [INFO] Tests run: 50, Failures: 0, Errors: 0, Skipped: 0
-[INFO] 
+[INFO]
 [INFO] ------------------------------------------------------------------------
 [INFO] BUILD SUCCESS
 [INFO] ------------------------------------------------------------------------
-[INFO] Total time:  2.401 s
-[INFO] Finished at: 2020-05-18T12:40:07-05:00
+[INFO] Total time:  2.276 s
+[INFO] Finished at: 2020-07-06T22:47:24-05:00
```

Comment:
TCK does not comform with the requirement noted in compatibility-certification-request.md. There is no Java SE info, no tck zip, etc.